manifest/src/lib.rs

```diff
@@ -0,0 +1,106 @@
+use anyhow::Result;
+use npm_package_json::Package;
+use std::collections::HashMap;
+use std::path::{Path, PathBuf};
+use std::{ffi::OsStr, fs};
+use strum::{Display, EnumIter};
+
+#[macro_use]
+extern crate lazy_static;
+
+lazy_static! {
```

Comment:
I'd replace this with a function and a `match` on `&str`. 

If it really has to be a static map, I think [`OnceCell::sync::Lazy`](https://docs.rs/once_cell/1.16.0/once_cell/#lazy-initialized-global-data) is the standard (now).

Reply:
a function it is 👍